jenkins/vendor_images.yaml

```diff
@@ -1,3 +1,3 @@
 docker.io:
   images-by-tag-regex:
-    jenkins/jenkins: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
\ No newline at end of file
+    jenkins/jenkins: ^v?2\.[0-9]+?$
```

Comment:
I think this will not properly vendor patch versions right?

Reply:
@michaeljguarino 
This will match 2.397 or 2.387. Check at https://regex101.com/

https://hub.docker.com/layers/jenkins/jenkins/2.397/images/sha256-4875abf22c4e465d723f5cb742218807f55d7772157ed62466c5cb45c0f9f755?context=explore

Reply:
From what I can see they do not have images with the patch version defined. Do we need to also vendor all the `2.XXX-jdk` images? They provide a lot of custom tags in addition to raw `2.XXX` tags.

Reply:
They do have semvers for patches. See [2.387.2(https://hub.docker.com/layers/jenkins/jenkins/2.387.2/images/sha256-8f7043722b3bb576fde60fa4ab59465a4b77e677c92774514897301ab77825a3?context=explore)